### PR TITLE
Fix the multiple-statements issue on pypy 3.8

### PR DESCRIPTION
**Pull request by Pierre-Sassoulas**
## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :bug: Bug fix          |

## Description

Problem appeared recently hard to know exactly why because we were using a cached environment for some time before.

### CODE REVIEW

File: tests/functional/a/alternative/alternative_union_syntax_error.py [hunk 1]
```diff
@@ -1,10 +1,18 @@
-"""Test PEP 604 - Alternative Union syntax
-without postponed evaluation of annotations.
+"""Test PEP 604 - Alternative Union syntax with postponed evaluation of
+annotations enabled.
 
-For Python 3.7 - 3.9: Everything should fail.
+For Python 3.7 - 3.9: Most things should work.
```
Comment (Pierre-Sassoulas) [inline tests/functional/a/alternative/alternative_union_syntax_error.py:4]:
```suggestion
For Python 3.7 - 3.9: Everything should fail.
```

**Review by DanielNoord — COMMENTED** [review]
I don't really understand this PR? Are all these formatting changes necessary for PyPy to pass?

**Comment by Pierre-Sassoulas** [comment]
I formatted the disable because as I added 3 new lines I had to regenerate the output anyway, might as well clean up a little.

**Comment by DanielNoord** [comment]
But there are also multiple file moves right? What's is the actual fix the unblock CI?

**Comment by Pierre-Sassoulas** [comment]
[814ae24](https://github.com/PyCQA/pylint/pull/7945/commits/814ae244a991f7c735e775ed9edc462eeb2c0166) (There might be little bit of fix in the moved files from the two previous commit which is why I did not separate them).

**Review by DanielNoord — APPROVED** [review]

**Pull request merged**